### PR TITLE
Troubleshooting entry for RN autolinking

### PR DESCRIPTION
**Pull request by marandaneto**
https://github.com/getsentry/sentry-react-native/issues/2267

### CODE REVIEW

File: src/platforms/react-native/troubleshooting.mdx [hunk 1]
```diff
@@ -168,3 +168,49 @@ If you're using `React Navigation 5` and you're not receiving transactions anymo
 ## RAM Bundles Mismatched Line Numbers
 
 If you experience mismatched line numbers on [sentry.io](https://sentry.io) when using RAM Bundles, this is due to a [bug](https://github.com/facebook/metro/issues/399) on the Metro tooling.
+
+## React Native 0.69 and higher
+
+If you're using React Native v0.69 or higher, the autolink has changed and we're working on a fix, follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2267).
+
+### Android
```
Comment (romtsn) [inline src/platforms/react-native/troubleshooting.mdx:176]: Is it only relevant for android? cus I see we changed ios config below as well

Comment (marandaneto) [reply src/platforms/react-native/troubleshooting.mdx:176]: It is for both, still a draft, figuring out iOS still.

File: src/platforms/react-native/troubleshooting.mdx [hunk 2]
```diff
@@ -168,3 +168,64 @@ If you're using `React Navigation 5` and you're not receiving transactions anymo
...
+};
+```
+
+This should work on Android, however, on iOS, it'd still fail in the `Bundle React Native code and images` build step phase and for that, you've to revert the patched `shellScript` in the `ios/$projectName.xcodeproj/project.pbxproj` file where `$projectName` is likely your project's name.
+Search for the build step phase `Bundle React Native code and images` and patch the `shellScript`.
+
+_Old_:
+
+```bash
+shellScript = "export SENTRY_PROPERTIES=sentry.properties\nexport EXTRA_PACKAGER_ARGS=\"--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map\"\nset -e\n\nWITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"\n../node_modules/@sentry/cli/bin/sentry-cli react-native xcode REACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"\n\n/bin/sh -c \"$WITH_ENVIRONMENT $REACT_NATIVE_XCODE\"\n";
```
Comment (marandaneto) [inline src/platforms/react-native/troubleshooting.mdx:222]: Or:
`shellScript = "set -e\n\nexport NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh\n";`
If RN is lower than 0.69 and not patched by Sentry yet.

**Comment by marandaneto** [comment]
@bruno-garcia please review and merge it if's ok, it's a workaround for now.

**Review by imatwawana — APPROVED** [review]
Made some styling and wording edits

**Comment by marandaneto** [comment]
> Made some styling and wording edits

Thank you @imatwawana 🌮

**Pull request merged**